bigbone/src/main/kotlin/social/bigbone/api/entity/Notification.kt

```diff
@@ -48,10 +48,18 @@ data class Notification(
     /**
      * Specifies the notification type.
      */
-    enum class Type(val value: String) {
-        Mention("mention"),
-        Reblog("reblog"),
-        Favourite("favourite"),
-        Follow("follow")
+    @Serializable
+    enum class Type {
```

Comment:
We should rename this to `NotificationType` (as you did similar in other places).